branch-config.mk.j2

```diff
@@ -41,13 +41,20 @@ L10N_DIR ?= f{$ distro_release $}
 BASE_CONTAINER ?= registry.fedoraproject.org/fedora:{$ distro_release $}
 COPR_REPO ?= \@rhinstaller/Anaconda-devel
 
-{% elif distro_name == "rhel" and distro_release != 10 %}
+{% elif distro_name == "rhel" and (distro_release != 10 or distro_release != 9) %}
 
 GIT_BRANCH ?= rhel-{$ distro_release $}
 L10N_DIR ?= rhel-{$ distro_release $}
 BASE_CONTAINER ?= registry-proxy.engineering.redhat.com/rh-osbs/ubi{$ distro_release $}:latest
 COPR_REPO ?= rhinstaller-group/Anaconda
 
+{% elif distro_name == "rhel" and distro_release == 9 %}
+
+GIT_BRANCH ?= rhel-{$ distro_release $}
+L10N_DIR ?= rhel-{$ distro_release $}
+BASE_CONTAINER ?= quay.io/centos/centos:stream9
+COPR_REPO ?= rhinstaller-group/Anaconda
```

Comment:
Please leave this as it was in before and change the `BASE_CONTAINER` link to the 

```
BASE_CONTAINER ?= quay.io/centos/centos:stream{$ distro_release $}
```

In the future we will remove also the condition for rhel-10, however, currently there are not final containers available which should be on the same URL